src/components/FormFields.tsx

```diff
@@ -36,42 +41,51 @@ export const TextField = createRestyleComponent<Props, Theme>(
   TextInput,
 )
 
-type LabeledTextField = {
+type FormFieldProps = BoxProps<Theme> & {
   label: string
-  value: string
+  value: any
```

Comment:
**Question:** this be avoided?

Reply:
When your solving a problem hackily, yes but because we're reusing this type multiple times with a range of value types , I don't consider it terrible. I will replace it with `string | number` as it's *more* correct.